tests/test_compressors/test_int_quant.py

```diff
@@ -57,7 +57,7 @@ def get_dummy_quant_config(strategy, group_size=None):
         [
             QuantizationStrategy.CHANNEL,
             False,
-            128,
+            None,
```

Comment:
why were the group sizes in these cases before?

Reply:
git blame @Satrat

Reply:
Not sure, but they were ignored before so I don't think it matters